pkg/kubernetes/kubernetes.go

```diff
@@ -13,6 +13,7 @@ import (
 	"k8s.io/client-go/discovery/cached/memory"
 	"k8s.io/client-go/dynamic"
 	"k8s.io/client-go/kubernetes"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/oidc"
```

Comment:
nit: this should be at the end of all imports, in order to emphasize it is in here just for dependencies.